.chezmoi.toml.tmpl

```diff
@@ -9,3 +9,9 @@
     sans-serif = '"Avenir Next","Avenir","Helvetica Neue","Helvetica",sans-serif"'
     serif = '"Georgia",serif'
     mono = '"Fira Code Mono"'
+
+[[textconv]]
+# https://www.chezmoi.io/user-guide/tools/diff/#show-human-friendly-diffs-for-binary-files
+pattern = "**/*.plist"
+command = "plutil"
+args = ["-convert", "xml1", "-o", "-", "-"]
```

Comment:
This allows me to `cm diff` and see changes to plist, which are simply XML files.